runtime.md

```diff
@@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
 See the specification of the `config.json` file for the definition of these fields.
 The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
 In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
-
-## Hooks
-
-Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
```

Comment:
I'd prefer if we removed all hooks, or by "post-stop hook" did you just mean "after they deleted the sub-container" ?

Either way, doesn't that feel a bit cumbersome just to do something between stop() and delete()?